doc/install/install.md

```diff
@@ -1,32 +1,120 @@
-# Installing OLM
+Installing OLM
+------
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+<!--**Table of Contents**   *generated with [DocToc](https://github.com/thlorenz/doctoc)*  -->
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Installation Options](#installation-options)
+  - [Release binary](#release-binary)
+  - [Minikube](#minikube)
+  - [Minishift](#minishift)
+  - [OpenShift](#openshift)
+    - [By OCP 3.11 installation scripts](#by-ocp-311-installation-scripts)
+    - [OCP 3.11](#ocp-311)
+- [Customizing OLM installation](#customizing-olm-installation)
+- [Uninstall](#uninstall)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Overview
 
 OLM deployment resources are templated so that they can be easily configured for different deployment environments.
 
+# Prerequisites  
+
+Installing the CRDs first gives them a chance to register before installing the rest, which requires the CRDs exist.
+
+```bash
+$ make install-crd
+```
+
+**IMPORTANT** Required to any manually installation 
+
+# Installation Options
+
+## Release binary
+
 Check out the latest [releases on github](https://github.com/operator-framework/operator-lifecycle-manager/releases) for release-specific install instructions.
 
-## Manual installation 
+## Minikube 
+
+This command starts minikube, builds the OLM containers locally with the minikube-provided docker, and uses the local configuration in [local-values.yaml](local-values.yaml) to build localized deployment resources for OLM.
+
+```bash
+$ make run-local
+```
+
+You can verify that the OLM components have been successfully deployed by running `kubectl -n local get deployments`
+
+**NOTE** It is recommended for development pruposed and will use the source locally
+
+## Minishift
```

Comment:
Minishift is deprecated, we don't want to steer users towards it

Reply:
Deprecated for OLM? Where it was deprecated? How to test it locally with OCP?

Reply:
I added the info that it is deprecated. I think we should keep it for a while